src/dialogs/FilingSurveyDialog.vue

```diff
@@ -0,0 +1,92 @@
+<template>
+  <v-dialog v-model="dialog" width="45rem" persistent :attach="attach" content-class="filing-survey-dialog">
+    <v-card class="px-10 py-6">
+      <v-card-title class="d-flex justify-space-between align-center">
+        <span class="font-weight-bold">Filing Survey</span>
+        <v-btn icon large class="dialog-close" @click="no()"><v-icon>mdi-close</v-icon></v-btn>
+      </v-card-title>
+
+      <v-card-text>
+        <p>
+          Do you want to be part of our continuous improvement process by sharing
+          your experience? A two-minute survey will appear in a new tab for you to
+          share your filing experience.
+        </p>
+        <p>
+          This information can really help us improve our services.
+        </p>
+
+        <p>
+          Our <a href="https://www2.gov.bc.ca/gov/content/home/privacy" target="_blank">Privacy Statement</a>
+        </p>
+
+        <v-checkbox
+          id="dialog-checkbox"
+          hide-details
+          label="Do not show this message again"
+          class="mt-6"
+          @change="updateCookie($event)"
+        />
+      </v-card-text>
+
+      <v-card-actions class="d-flex justify-center">
+        <v-btn
+          id="dialog-no-button"
+          large outlined color="primary"
+          class="mr-sm-3 px-8"
```

Comment:
A bit of right margin to space this button from the one on the right on screens larger than "sm". On narrower screens, this margin is not applied and the buttons are closer together, ie:

![image](https://user-images.githubusercontent.com/10002889/213829279-226d4e0e-51e8-43cb-a5fc-f5411ccd6881.png)